src/main/fc/fc_tasks.c

```diff
@@ -586,6 +592,15 @@ cfTask_t cfTasks[TASK_COUNT] = {
     },
 #endif
 
+#ifdef RUNCAM_SPLIT_SUPPORT
+    [TASK_RCSPLIT_SUPPORT] = {
+        .taskName = "RCSPLIT",
+        .taskFunc = rcsplitProcess,
+        .desiredPeriod = TASK_PERIOD_HZ(100),        // 100 Hz, 10ms
```

Comment:
Oh, I'm sorry I am not very familiar with RC mode activation, before start using RC mode activation, my goal is control our camera through remote control, so I'v ask in the developer channel in cf slack, and study the code in cleanflight, and finally  use  RC mode activation to implement this feature.